test/test-pages/bbc-1/expected-metadata.json

```diff
@@ -1,7 +1,6 @@
 {
   "title": "Obama admits US gun laws are his 'biggest frustration' - BBC News",
   "byline": null,
-  "dir": "ltr",
```

Comment:
Remove this `dir` for two reason.

1. This `dir` should not be existed before because we don't do text direction test before the patch.
2. If we want to keep this, the value of it would be `null` not `ltr`. Because there is no `dir` attribute in the `bbc-1` test page.